src/v3/features/ministry/add_service_time_modal_window/index.tsx

```diff
@@ -0,0 +1,325 @@
+import { Box, TextField } from '@mui/material';
+
+import { StyledBox, StyledModalWindowContainer, StyledRowContainer } from './add_service_time_modal_window.styled';
+import { AddServiceTimeModalWindowProps } from './add_service_time_modal_window.types';
+import useAppTranslation from '@hooks/useAppTranslation';
+
+import CustomTypography from '@components/typography';
+import DatePicker from '@components/date_picker';
+import MinusButton from '@components/minus_button';
+import PlusButton from '@components/plus_button';
+import Button from '@components/button';
+import { useContext, useEffect, useRef, useState } from 'react';
+import { CustomDropdownContainer, CustomDropdownItem, CustomDropdownMenu } from '@components/dropdown';
+import { IconAdd, IconLanguageCourse, IconPersonalDay, IconSchool, IconSchoolForEvangelizers } from '@components/icons';
+import { hoursToSeconds } from 'date-fns';
+import { EditAndAddBibleStudyContext } from '../EditAndAddBibleStudyContext';
+
+import { SHA256 } from 'crypto-es/lib/sha256';
```

Comment:
@FussuChalice: could we please use the default `crypto.randomUUID()` for this purpose, instead of using this library?